solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -1000,6 +1002,26 @@ private void loadInternal() {
 
     // initialize gauges for reporting the number of cores and disk total/free
 
+    Attributes containerAttrs =
+        Attributes.builder().put(CATEGORY_ATTR, SolrInfoBean.Category.CONTAINER.toString()).build();
+
+    solrMetricsContext.observableLongGauge(
+        "solr_cores_loaded",
```

Comment:
I forget... are we namespacing all metrics under major areas (as we are doing for system props / env) lately?  If so, a name like "solr_node_cores_ ..." would be better.  If not... ok.